tests/datetime_test.py

```diff
@@ -34,6 +48,33 @@ def setUp(self):
         self.tdvec2 = ak.Timedelta(ak.ones(100, dtype=ak.int64), unit="s")
         self.onesecond = pd.Timedelta(1, unit="s")
 
+    def test_timedelta(self):
+        ak_df = build_ak_df()
+        td1 = ak.timedelta_range(start=0, periods=10, freq="s")
+        td2 = ak.Timedelta(ak.arange(10), unit="s")
+        self.assertTrue(isinstance(td1, ak.Timedelta) and isinstance(td2, ak.Timedelta))
+        self.assertTrue(td1.components.equals(ak_df.to_pandas()))
+        self.assertTrue(((td1.components == td2.components).all()).all())
+
+    def test_datetime_accessors(self):
+        start_time = "2022-12-31 23:59:55"
+        tdvec3 = ak.date_range(start=start_time, periods=10, freq="s")
+
+        exp = tdvec3.to_pandas()
+        exp_date = exp.floor('d')
+        self.assertTrue(((tdvec3.date == exp_date).all()).all())
+        self.assertTrue(((tdvec3.year == ak.array(exp.year)).all()).all())
+        self.assertTrue(((tdvec3.month.to_ndarray() == exp.month).all()).all())
+        self.assertTrue(((tdvec3.day.to_ndarray() == exp.day).all()).all())
+        self.assertTrue(((tdvec3.hour.to_ndarray() == exp.hour).all()).all())
+        self.assertTrue(((tdvec3.minute.to_ndarray() == exp.minute).all()).all())
+        self.assertTrue(((tdvec3.second.to_ndarray() == exp.second).all()).all())
+        self.assertTrue(((tdvec3.dayofweek.to_ndarray() == exp.dayofweek).all()).all())
+        self.assertTrue(((tdvec3.dayofyear.to_ndarray() == exp.dayofyear).all()).all())
+        actual_total_seconds = list(tdvec3.total_seconds.to_ndarray())
+        exp_total_seconds = [np.datetime64(x,'s').astype(float) for x in np.array(exp)]
+        self.assertTrue(actual_total_seconds == exp_total_seconds)
```

Comment:
This is testing that to_pandas works because you are converting the ak object to pandas and then testing that against the result. Build exp directly in pandas, not from the arkouda object.

Reply:
Now testing against a pandas data_range